src/ClientProtos/capture_data.proto

```diff
@@ -164,6 +167,36 @@ message TimerInfo {
   uint64 timeline_hash = 11;
   repeated uint64 registers = 12;
   Color color = 13;
+  uint64 group_id = 14;
+  uint64 id = 15;
+  uint64 address_in_function = 16;
+  string name = 17;
```

Comment:
Should `id` and `name` say a bit more? For example, there's already `callstack_id` and `function_id` and just `id` would be confusing.

Reply:
I changed `id` to `async_scope_id`. I think we should keep the `name`, as it will be used for both new types.

Reply:
> I think we should keep the name, as it will be used for both new types.

Which types exactly? Could we maybe find a common name but still more than `name`?

Reply:
It would be `ApiScope`, where the name is the name of the scope and `ApiScopeAsync` where the name is essentially the name of the track.

Reply:
`api_scope_name`? It's still somehow the name of the scope for the async scopes, just that we group them by name.

Reply:
Done.